include/revng/Pipeline/Context.h

```diff
@@ -96,6 +101,28 @@ class Context {
 
   const GlobalsMap &getGlobals() const { return Globals; }
 
+  void addWriteOnceContainer(llvm::StringRef Name,
```

Comment:
Can we come up with a name that's more intuitive than "WriteOnceContainer"?
Maybe "ConstantContainer" or "GlobalContainer" (sad).